src/components/Header/Header.module.scss

```diff
@@ -0,0 +1,19 @@
+@import url('https://fonts.googleapis.com/css2?family=KoHo&display=swap');
```

Comment:
Be careful with this. I am not sure @import of external resource is a good ideal -  https://gtmetrix.com/avoid-css-import.html. 

I know I have just mentioned @import in the above comment 
![CleanShot 2023-10-06 at 15 06 42](https://github.com/KamilaSemotanova/tracker/assets/12488648/54c75468-8ebd-4968-b650-cc418bad12f1)
and feel free to let me know if it is possible to import the constants/colors with @use I would use it on my projects!

but this is an external resource. It will slow down the page loading. It might cause FOUC - a flash of unstyled content. 

Always prefer local fonts. Yes, even if it means literally downloading the font files. The cache files are usually well cached, but I would choose local day and night. It will have to be sent to the client regardless. 

Use either font-face or in the case of next.js their documented approach 
- https://nextjs.org/docs/pages/building-your-application/optimizing/fonts
- https://nextjs.org/docs/app/api-reference/components/font